### ISSUE TITLE
No indent warning for chained functions on a new line.

### ISSUE THREAD

**Issue by alasdairhurst** [eslint/eslint#8222]
**Tell us about your environment**

* **ESLint Version:** 3.17.1
* **Node Version:** 6.2.2
* **npm Version:** 3.9.5

**What parser (default, Babel-ESLint, etc.) are you using?**
 babel-eslint 6.1.2

**Please show your full configuration:**

```
{
	"parser": "babel-eslint",
	"extends": "eslint-config-airbnb-base",
	"env": {
		"browser": true,
		"node": true
	},
	"sourceType": "module",
	"plugins": [
		"import"
	],
	"rules": {
			"quotes": [ 1, "single" ],
			"no-mixed-spaces-and-tabs": "error",
			"comma-dangle": [ 1, "never"],
			"indent": ["error", "tab"],
			"func-names": 0,
			"no-underscore-dangle" : 0,
			"no-console": ["error", { "allow": ["warn", "error"] }],
			"prefer-const" : 1,
			"prefer-arrow-callback" : 0,
			"no-param-reassign" : 0,
			"class-methods-use-this" : 0,
			"prefer-template" : 1,
			"no-cond-assign" : 1,
			"no-useless-escape" : 1,
			"object-shorthand": 0,
			"no-prototype-builtins" : 0,
			"quote-props" : 1,
			"no-tabs": "off",
			"one-var": "off",
			"arrow-parens": "off",
			"semi": 1,
			"no-plusplus": 0,
			"import/no-extraneous-dependencies": "off",
			"import/prefer-default-export": "off",
			"import/no-named-as-default": "off",
			"no-multi-assign": "off",
			"no-restricted-syntax": "off",
			"import/no-dynamic-require": "off"
		}
}
```

**What did you do? Please include the actual source code causing the issue.**

```js
const obj = { fn: () => {} };
obj
                              .fn();

```

**What did you expect to happen?**
Warning - saw 30 spaces and expected 1 tab
**What actually happened? Please include the actual, raw output from ESLint.**
Nothing. Nada. This happens with any number of function chains and i can mix and match lines with leading tabs or spaces.

**Comment by gyandeeps** [comment]
You have to activate the MemberExpression option on indent rule.
Docs: http://eslint.org/docs/rules/indent#memberexpression

**Comment by vitorbal** [comment]
@alasdairhurst does @gyandeeps 's suggestion above solve your problem?

**Comment by alberto** [comment]
Closing this issue as it looks like the question has been answered. Please feel free to visit us in the ESLint Gitter if you have any other issues!

**Comment by alasdairhurst** [comment]
Sorry, @vitorbal @gyandeeps  Yes, it fixed it, thanks!
